Plugins/Creature/Creature.cpp

```diff
@@ -1047,12 +1047,27 @@ NWNX_EXPORT ArgumentStack SetClassByPosition(ArgumentStack&& args)
     {
         const auto position = args.extract<int32_t>();
         const auto classID = args.extract<int32_t>();
+        const auto bUpdateLevels = args.extract<int32_t>();
```

Comment:
If you want to be extra fancy you can replace the above with: 

```cpp
int32_t bUpdateLevels = true;
try
{
    bUpdateLevels = args.extract<int32_t>()
}
catch(const std::runtime_error& e)
{
    LOG_WARNING("NWNX_Creature_SetClassByPosition() called from NWScript without bUpdateLevels parameter. Please update nwnx_creature.nss");
}
```

So it doesn't throw when someone calls the function without recompiling their script.

Reply:
Let them ~~eat cake~~ read changelogs.